admin/tests/integration/components/user-detail-personal-information-test.js

```diff
@@ -398,4 +398,106 @@ module('Integration | Component | user-detail-personal-information', function(ho
     });
   });
 
+  module('when the administrator click on remove authentication method button', function() {
+
+    test('should display remove authentication methode confirm modal', async function(assert) {
```

Comment:
`    test('should display remove authentication method confirm modal', async function(assert) {`

Reply:
Done